doc/psyclone_script.rst

```diff
@@ -0,0 +1,257 @@
+    --profile {invokes,kernels}, -p {invokes,kernels}
+                          Add profiling hooks for either 'kernels' or 'invokes'
+    --force-profile {invokes,kernels}
+                          Add profiling hooks for either 'kernels' or 'invokes'
+                          even if a transformation script is used. Use at your
+                          own risk.
+    -v, --version         Display version information (1.6.0)
+
+Basic Use
+---------
+
+The simplest way to use ``psyclone`` is to provide it with an
+algorithm file::
+
+    > psyclone alg.f90
+
+If the algorithm file is invalid for some reason, the script should
+return with an appropriate error. For example, if we use the Python
+``genkernelstub`` script as an algorithm file we get the following::
+
+    > psyclone <PSYCLONEHOME>/bin/genkernelstub
+    ...
+        1:#!/usr/bin/env python <== no parse pattern found for "#" in 'BeginSource' block.
+    'Parse Error: Fatal error in external fparser tool'
+
+If the algorithm file is valid then the modified algorithm code and
+the generated PSy code will be output to the terminal screen.
+
+
+Choosing the API
+----------------
+
+In the previous section we relied on PSyclone using the default
+API. The default API, along with the supported API's can be seen by
+running the ``psyclone`` script with the ``-h`` option.
+
+If you use a particular API frequently and it is not the default then
+you can change the default by creating a copy of the default
+``psyclone.cfg`` file and editing it. See :ref:`configuration` for
+more details.
+
+If your code uses an API that is different to the default then you can
+specify this as an argument to the ``psyclone`` script.
+::
+
+    > psyclone -api dynamo0.1 alg.f90
+
+File output
+-----------
+
+By default the modified algorithm code and the generated PSy code are
+output to the terminal. These can instead be output to files by using the
+``-oalg <file>`` and ``-opsy <file>`` options, respectively. For example, the
+following will output the generated psy code to the file 'psy.f90' but
+the algorithm code will be output to the terminal:
+::
+
+    > psyclone -opsy psy.f90 alg.f90
+
+Algorithm files with no invokes
+-------------------------------
+
+If the ``psyclone`` script is provided with a file that contains no
+``invoke`` calls then the script outputs a warning to ``stdout`` and
+copies the input file to ``stdout``, or to the specified algorithm
+file (if the ``-oalg <file>`` option is used). No PSy code will be
+output. If a file is specified using the ``-opsy <file>`` option this file
+will not be created.
+
+.. code-block:: bash
+
+    > psyclone -opsy psy.f90 -oalg alg_new.f90 empty_alg.f90
+    Warning: 'Algorithm Error: Algorithm file contains no invoke() calls: refusing to
+    generate empty PSy code'
+
+Kernel directory
+----------------
+
+When an algorithm file is parsed, the parser looks for the associated
+kernel files. The way in which this is done requires that any
```

Comment:
Capitalisation of "Kernel" is also inconsistent throughout the section.

Reply:
I've gone through and changed it to "kernel" - thanks.